### PR TITLE
US105635 -- Fixed hardcoded text

### PR DESCRIPTION
**Pull request by mpharoah-d2l**
LMS side: https://git.dev.d2l/projects/CORE/repos/platformtools/pull-requests/3089/overview

### CODE REVIEW

File: editor/d2l-rubric-criterion-editor.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ $_documentContainer.innerHTML = /*html*/`<dom-module id="d2l-rubric-criterion-ed
 
 		<simple-overlay id="overlay" tabindex="-1" scroll-action="lock" class="with-margin scrollable"  with-backdrop>
 			<div>
-				<h2>Browse Outcomes </h2>
+				<h2>[[browseOutcomesText]]</h2>
```
Comment (djwester) [inline editor/d2l-rubric-criterion-editor.js]: can't we just keep the 'Browse' and replace the 'outcomes' part of this with [[outcomesTitle]]. So something like:

`<h2> Browse [[outcomesTitle]]</h2>`

Comment (mpharoah-d2l) [reply editor/d2l-rubric-criterion-editor.js]: Because then we'd end up with things like 'Browse 標準'

Comment (djwester) [reply editor/d2l-rubric-criterion-editor.js]: well, yeah, but can't we localize Browse?

Comment (mpharoah-d2l) [reply editor/d2l-rubric-criterion-editor.js]: Word order isn't necessarily going to be consistent across languages

**Pull request merged**